v3/src/lib/use-cloud-file-manager.ts

```diff
@@ -19,22 +141,23 @@ export function useCloudFileManager(optionsArg: CFMAppOptions) {
 
     const _options: CFMAppOptions = {
       ui: {
-        // menuBar: {
-        //   info: "Language menu",
-        //   languageMenu: {
-        //     currentLang: "EN-us",
-        //     options: DG.locales.map(function (locale) {
-        //       return {
-        //         label: locale.langName,
-        //         langCode: locale.langDigraph,
-        //       };
-        //     }),
-        //     onLangChanged: function (langCode) {
-        //       DG.log('Changed language: ' + langCode);
-        //       window.location = makeNewLocaleUrl(langCode, window.location);
-        //     }
-        //   }
-        // },
+        menuBar: {
+          info: "Language menu",
+          languageMenu: {
+            currentLang: "EN-us",
+            options: locales.map(function (locale) {
+              return {
+                label: locale.langName,
+                langCode: locale.langDigraph,
+              }
+            }),
+          },
+          onLangChanged: (langCode: string) => {
+            Logger.log(`Changed language: ${langCode}`)
+            window.location.href = makeNewLocaleUrl(langCode, window.location.href)
```

Comment:
The need to reload the page when switching languages in v2 was an artifact of SproutCore's approach to localization. In v3 we're just going to change some state and force everything to re-render. You can just remove the `makeNewLocaleUrl` function and make this PR UI only for now.